packages/itinerary-body/src/TransitLegBody/alerts-body.tsx

```diff
@@ -131,6 +134,24 @@ export default function AlertsBody({
                       }}
                     />
                   )}
+                  {alertUrl.trim() && (
+                    <S.TransitAlertExternalLink href={alertUrl} target="_blank">
+                      <ExternalLinkAlt height={10} />
```

Comment:
We shouldn't specify a height here? Or if we do we should flex align to baseline so that it is better aligned